shadow-dom/untriaged/events/events-created-by-users-do-not-stop/test-001.html

```diff
@@ -40,20 +40,22 @@
     inp1.setAttribute('id', 'inp1');
     inp1.setAttribute('type', 'checkbox');
     s.appendChild(inp1);
+    var pass = false;
 
     s.addEventListener('abort', A_05_04_01_T01.step_func(function(event) {
-    	assert_equals(event.target.getAttribute('id'), 'inp1', 'Inside shadoe tree: Wrong target');
+    	assert_equals(event.target.getAttribute('id'), 'inp1', 'Inside shadow tree: Wrong target');
```

Comment:
It seems you introduced a "TAB" on this line for all files.
Could you convert to spaces?